weave/cookbooks/source/Intro_to_Weave_Hello_Eval.ipynb

```diff
@@ -56,124 +53,187 @@
    },
    "outputs": [],
    "source": [
-    "# Install dependancies and imports\n",
-    "!pip install wandb weave openai -q\n",
+    "# Find your wandb API key at: https://wandb.ai/authorize\n",
+    "# Ensure that your wandb API key is available at:\n",
```

Comment:
A lot of the notebooks handle the setup differently in a way that allows you to paste it into the notebook instead of needing to set it in the environment. See https://github.com/wandb/examples/blob/7323cc40bec88074ad7dcc8bbf27948166571739/weave/docs/quickstart_dspy.ipynb#L136. This might be useful.